bin/juxtaposer/deploy/bootstrap.sh

```diff
@@ -1,16 +1,16 @@
-export CONFIG_TEMPLATE=pg
+export CONFIG_TEMPLATE=CHANGEME
 export TEST_DURATION=1h
 
 export APP_NAME=juxtaposer-${CONFIG_TEMPLATE}
 
 export APP_SERVICE_ACCOUNT=secretless-xa
-export AUTHENTICATOR_ID=openshift/xa-secretless
-export DAP_ACCOUNT=xa
-export DAP_NAMESPACE_NAME=xa-secretless
+export AUTHENTICATOR_ID=CHANGEME
+export DAP_ACCOUNT=CHANGEME
+export DAP_NAMESPACE_NAME=CHANGEME
 export DAP_SSL_CERT_CONFIG_MAP=dap-ssl-cert
-export DOCKER_REGISTRY_PATH=REPLACEME
-export SECRETELESS_IMAGE="cyberark/secretless-broker:1.2.0"
-export TEST_APP_NAMESPACE_NAME=srdjan-secretless-xa
+export DOCKER_REGISTRY_PATH=CHANGEME
+export SECRETLESS_IMAGE=CHANGEME
+export TEST_APP_NAMESPACE_NAME=CHANGEME
```

Comment:
I would leave comments with example comments of what these changed vars maybe should look like though